# 現在の情報を取得する(SCIBOT-Xのみ)
---

